<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaitScreenGm</class>
 <widget class="QDialog" name="WaitScreenGm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku Warriors</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Imagens/sudokuicon3.png</normaloff>../../Imagens/sudokuicon3.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3498db;</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Numero de jogadores na sala:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>ESPERNADO OUTROS JOGADORES</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="wait_points">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>60</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>. . . . .</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: gold; border: 2px solid gray; border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Iniciar Jogo</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_atual_players_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
